frontend/src/metabase/qb/components/drill/CountByColumnDrill.js

```diff
@@ -0,0 +1,46 @@
+/* @flow */
+
+import React from "react";
+
+import {
+    summarize,
+    pivot,
+    getFieldClauseFromCol
+} from "metabase/qb/lib/actions";
+import * as Card from "metabase/meta/Card";
+import { isCategory } from "metabase/lib/schema_metadata";
+
+import type {
+    ClickAction,
+    ClickActionProps
+} from "metabase/meta/types/Visualization";
+
+export default (
+    { card, tableMetadata, clicked }: ClickActionProps
+): ClickAction[] => {
+    const query = Card.getQuery(card);
+
+    if (
+        !query ||
+        !clicked ||
+        !clicked.column ||
```

Comment:
Would be nice to destructure the values with something like [Ramda's pick method](http://ramdajs.com/docs/#pick), means that we can use 1-2 conditions instead of the current four conditions regarding `clicked`